01-HTML-basics/exercise1_html/signup.html

```diff
@@ -3,6 +3,9 @@
 	<head>
   		<meta name="description" content="Signup page">
  		<meta name="author" content="Ezequiel Vannucci">
+ 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+ 		<title>Signup</title>
 	</head>
 
 
```

Comment:
This form has 2 tags with the functionalyti `submit`, there's an input in line 25 and a button in line 39, there's should be just 1 of them.